automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebooksListPage.scala

```diff
@@ -7,6 +7,26 @@ import org.openqa.selenium.WebDriver
 
 import scala.util.Try
 
+trait Kernel {
```

Comment:
Minor, maybe make these `sealed trait` and `case object`s. That is useful if we're pattern matching on these things (which we're not currently doing, but still good practice).